apps/prairielearn/src/models/course.ts

```diff
@@ -40,6 +40,26 @@ export async function selectCourseById(course_id: string): Promise<Course> {
   );
 }
 
+export async function selectCourseIdByInstanceId(course_instance_id: string): Promise<string> {
+  return await queryRow(
+    sql.select_course_id_by_instance_id, 
+    { 
+      course_instance_id, 
+    },
+    IdSchema,
+  );
+}
```

Comment:
I don't see where this function is used either

Reply:
This is used to get the course id. The course id is needed for the navbar, which links to the page showing all public questions, and will later link to the public course overview page